api/management/api.go

```diff
@@ -343,6 +355,22 @@ type IdentityProviderRepresentation struct {
 	TrustEmail                *bool                   `json:"trustEmail,omitempty"`
 }
 
+// ThemeConfiguration struct
+type ThemeConfiguration struct {
+	Settings *configuration.ThemeConfigurationSettings `json:"settings,omitempty"`
+	Logo     *string                                   `json:"logo,omitempty"`
+	Favicon  *string                                   `json:"favicon,omitempty"`
+}
+
+// UpdatableThemeConfiguration struct
+type UpdatableThemeConfiguration struct {
+	ThemeName    *string                                   `json:"theme_name,omitempty"`
+	Settings     *configuration.ThemeConfigurationSettings `json:"settings,omitempty"`
+	Translations *map[string]any                           `json:"translations,omitempty"`
+	Logo         *[]byte                                   `json:"logo,omitempty"`
```

Comment:
slices can be nil... Don't use a pointer here, it adds complexity with indirections.
We used to use pointer but we are trying to remove them when not required